ngraph/core/src/op/gather.cpp

```diff
@@ -128,6 +128,166 @@ shared_ptr<Node> op::v1::Gather::clone_with_new_inputs(const OutputVector& new_a
     return make_shared<v1::Gather>(new_args.at(PARAMS), new_args.at(INDICES), new_args.at(AXIS));
 }
 
+NGRAPH_RTTI_DEFINITION(op::v7::Gather, "Gather", 7);
+static constexpr int64_t AXIS_NOT_SET_VALUE = std::numeric_limits<int64_t>::max();
```

Comment:
Please, remove this line

Reply:
done. will push together with other changes (if they are needed)

Reply:
done